modules/ROOT/pages/configure-monitoring-cloudhub.adoc

```diff
@@ -43,13 +45,13 @@ an appropriate value, for example:
 anypoint-cli --username="..." --password="..."
  --organization="Mythical, Inc"
  --environment="Production" runtime-mgr cloudhub-application deploy
- --runtime="3.9.0-AM" helloworld ~/demoproj2.zip
+ --runtime="3.9.4" helloworld ~/demoproj2.zip
 ----
 +
 * For Maven deployments:
 +
-If you are using Maven for deployment, you need to modify the `<muleVersion>`
-tag in the `pom.xml` file to append the `-AM`, for example, changing `<muleVersion>3.9.0</muleVersion>` to `<muleVersion>3.9.0-AM</muleVersion>`).
+If you are using Maven for deployment, you must modify the `<muleVersion>`
+tag in the `pom.xml` file. For example, changing `<muleVersion>3.9.3</muleVersion>` to `<muleVersion>3.9.4</muleVersion>`).
```

Comment:
You're not asking the reader to modify the tag; you're asking the reader to change the value of `muleVersion`.

Reply:
Which is a tag in the pom file.